crates/dslab-network/src/topology_structures.rs

```diff
@@ -7,14 +7,34 @@ pub type NodeLinksMap = BTreeMap<NodeId, BTreeMap<NodeId, LinkID>>;
 
 pub const INVALID_NODE_ID: usize = usize::MAX;
 
+#[derive(Copy, Clone, Debug)]
+pub enum LinkType {
+    Shared,
+    Fatpipe,
+}
```

Comment:
```suggestion
pub enum BandwidthSharingPolicy {
    Shared,
    NonShared,
}
```

Fatpipe это специфичный термин из SimGrid, предалагаю назвать более прямолинейно )